requirements_dev.txt

```diff
@@ -0,0 +1,4 @@
+-r requirements.txt
```

Comment:
oh, I remember this syntax, to be honest, I think, in general, this is a bad idea, but ok for now